proto/v2/skaffold.proto

```diff
@@ -174,7 +174,7 @@ message ApplicationLogEvent {
     string containerName = 1; // container that the log came from
     string podName = 2; // pod that the log came from
     string message = 3; // contents of the log
-    string combined_message = 4; // full message that skaffold writes, with format and color
+    string rich_formatted_message = 4; // full message that skaffold writes, with format and color
```

Comment:
Please use camelCase like `podName` and `containerName`

Reply:
oops, don't know why I did that 🤦